framework/test/unit/hooks/use-interval-test.ts

```diff
@@ -0,0 +1,20 @@
+import { renderHook } from '@testing-library/react'
+import { assert } from 'chai'
+import BluebirdPromise from 'bluebird'
+import { useLoadingMessage } from '../../../lib/hooks/use-loading-message'
+
+describe('useLoadingMessage', () => {
+  it('returns the value and adds a dot to it every 300 ms', async () => {
+    const { result } = renderHook(
+      (v) => useLoadingMessage(v),
+      { initialProps: 'loading' }
+    )
+
+    assert.equal(result.current, 'loading.')
+
+    await BluebirdPromise.delay(400)
```

Comment:
https://testing-library.com/docs/dom-testing-library/api-async/#waitfor could we use this?